### PR TITLE
Implement resend confirmation code method

### PR DESCRIPTION
**Pull request by imrobinized**
Added a new function `resend_confirmation_code` to Cognito object, which accept username as argument to trigger resend confirmation code.

I noted that there's no tests for confirmation. So I didn't add any.

I would like to make other changes if needed.

### CODE REVIEW

File: warrant/__init__.py [hunk 1]
```diff
@@ -346,6 +346,18 @@ def confirm_sign_up(self,confirmation_code,username=None):
         self._add_secret_hash(params, 'SecretHash')
         self.client.confirm_sign_up(**params)
 
+    def resend_confirmation_code(self,username=None):
```
Comment (armicron) [inline warrant/__init__.py]: It is better with a whitespace after the comma.

File: warrant/__init__.py [hunk 2]
```diff
@@ -346,6 +346,18 @@ def confirm_sign_up(self,confirmation_code,username=None):
...
+        """
+        Resend the confirmation for registration
+        :param username: User's username
+        """
+        if not username:
+            username = self.username
+        params = {'ClientId': self.client_id,
+                  'Username': username}
```
Comment (armicron) [inline warrant/__init__.py]: You can remove 2LOC, but it's up to you.
```python
'Username': self.username if username is None else username
```

Comment (imrobinized) [reply warrant/__init__.py]: Yeah of course, thanks for pointing out! Updated.

File: warrant/__init__.py (hunk 1 above)
Comment (imrobinized) [reply warrant/__init__.py]: Agree and updated, thanks!

**Pull request merged**